packages/react-app/src/hooks/useUrlFilters.ts

```diff
@@ -48,13 +54,29 @@ export const useDynamicUrl = (filters: FilterParams, ready: boolean): string =>
 	}, [filters, customer, debounceSearch, ready]);
 };
 
+const sanitizeFilter = (key: string, val: string): string | string[] => {
+	if (!keysWithArrayVals.includes(key) || typeof val !== 'string') return val;
+
+	const arrayVal: any[] = val.split(',').map((v: string) => v.trim());
+	switch (key) {
+	case 'status':
```

Comment:
Can you explain the intent of this functionality? we have other fliters in the url than just `status` and `paidStatus`, for instance `lte` and `gte` come to mind

Reply:
Right, I will sanitise it for `lte`, `gte` and `asc` as well